cupy/lib/_routines_poly.py

```diff
@@ -124,41 +182,51 @@ def polyfit(x, y, deg, rcond=None, full=False, w=None, cov=False):
     """Returns the least squares fit of polynomial of degree deg
     to the data y sampled at x.
 
-    Args:
-        x (cupy.ndarray): x-coordinates of the sample points of shape (M,).
-        y (cupy.ndarray): y-coordinates of the sample points of shape
-            (M,) or (M, K).
-        deg (int): degree of the fitting polynomial.
-        rcond (float, optional): relative condition number of the fit.
-            The default value is ``len(x) * eps``.
-        full (bool, optional): indicator of the return value nature.
-            When False (default), only the coefficients are returned.
-            When True, diagnostic information is also returned.
-        w (cupy.ndarray, optional): weights applied to the y-coordinates
-            of the sample points of shape (M,).
-        cov (bool or str, optional): if given, returns the coefficients
-            along with the covariance matrix.
-
-    Returns:
-        cupy.ndarray or tuple:
-        p (cupy.ndarray of shape (deg + 1,) or (deg + 1, K)):
-            Polynomial coefficients from highest to lowest degree
-        residuals, rank, singular_values, rcond \
-        (cupy.ndarray, int, cupy.ndarray, float):
-            Present only if ``full=True``.
-            Sum of squared residuals of the least-squares fit,
-            rank of the scaled Vandermonde coefficient matrix,
-            its singular values, and the specified value of ``rcond``.
-        V (cupy.ndarray of shape (M, M) or (M, M, K)):
-            Present only if ``full=False`` and ``cov=True``.
-            The covariance matrix of the polynomial coefficient estimates.
-
-    .. warning::
-
-        numpy.RankWarning: The rank of the coefficient matrix in the
-        least-squares fit is deficient. It is raised if ``full=False``.
-
-    .. seealso:: :func:`numpy.polyfit`
+    Parameters
+    ----------
+    x : cupy.ndarray
+        x-coordinates of the sample points of shape (M,)
+    y : cupy.ndarray
+        y-coordinates of the sample points of shape
+        (M,) or (M, K)
+    deg : int
+        Degree of the fitting polynomial
+    rcond : float, optional
+        Relative condition number of the fit.
+        The default value is ``len(x) * eps``.
+    full : bool, optional
+        Indicator of the return value nature.
+        When False (default), only the coefficients are returned.
+        When True, diagnostic information is also returned.
+    w : cupy.ndarray, optional
+        Weights applied to the y-coordinates
+        of the sample points of shape (M,).
+    cov : bool or str, optional
+        If given, returns the coefficients
+        along with the covariance matrix.
+
+    Returns
+    -------
+    p : cupy.ndarray of shape (deg + 1,) or (deg + 1, K)
+        Polynomial coefficients from highest to lowest degree
+    residuals, rank, singular_values, rcond :
+    cupy.ndarray, int, cupy.ndarray, float
```

Comment:
It seems that the documentation is not generated expectedly.
https://cupy--6469.org.readthedocs.build/en/6469/reference/generated/cupy.polyfit.html

Reply:
Yeah, thanks for pointing it out. I've updated it :)